src/02-common-principles.md

```diff
@@ -674,7 +674,89 @@ specificity and persistence.
 
 Several fields are designated for DOIs, for example, `DatasetDOI` in `dataset_description.json`.
 DOI values SHOULD be fully specified URIs such as `doi:10.18112/openneuro.ds000001.v1.0.0`.
-Bare DOIs such as `10.18112/openneuro.ds000001.v1.0.0` are [DEPRECATED][deprecated].
+Bare DOIs such as `10.18112/openneuro.ds000001.v1.0.0` are [DEPRECATED][].
+
+### BIDS URI
+
+To reference files in BIDS datasets, the following URI scheme may be used:
+
+```plain
+bids:[<dataset-name>]:<relative-path>
+```
+
+The scheme component `bids` identifies a BIDS URI,
+which defines a `path` component of the form `<dataset-name>:<absolute-path>`.
+The `dataset-name` component is an identifier for a BIDS dataset,
+and the `relative-path` component is the location of a resource within the
+BIDS dataset, relative to the root of that dataset.
+
+Examples:
+
+```plain
+bids::sub-01/fmap/sub-01_dir-AP_epi.nii.gz
+bids:ds000001:sub-02/anat/sub-02_T1w.nii.gz
+bids:myderivatives:sub-03/func/sub-03_task-rest_space-MNI152_bold.nii.gz
+```
+
+If no dataset name is specified, the URI is relative to the current BIDS dataset.
+This is made more precise in the next section.
+
+#### Resolution of BIDS URIs
+
+In order to resolve a BIDS URI, the dataset name must be mapped to a BIDS dataset.
+
+The special case `""` (that is, the empty string) refers to the BIDS dataset in
+which the BIDS URI is found.
+The dataset root is the nearest parent directory that contains a valid
+`dataset_description.json`.
+
+All other dataset names MUST be specified in the `DatasetLinks` object in
+[dataset_description.json][], which maps dataset names to URIs that point
+to BIDS dataset locations.
+If the scheme is omitted from a URI in `DatasetLinks`,
+that path is resolved relative to the current dataset root
+(see `deriv1` example, below).
+
+BIDS URIs cannot be interpreted outside a BIDS dataset,
+as they require a `dataset_description.json` file to resolve.
+
+#### Examples
+
+Consider this example `dataset_description.json`:
+
+```YAML
+{
+    ...
+    "DatasetLinks": {
+        "deriv1": "derivatives/derivative1",
+        "phantoms": "file:///data/phantoms",
+        "ds000001": "doi:10.18112/openneuro.ds000001.v1.0.0"
+    }
```

Comment:
What would an example look like for referencing a raw file from derivatives dataset nested in a raw BIDS dataset?
(will often be needed instead of the now deprecated `RawSources`)

Like this?

```yaml
{
    ...
    "DatasetLinks": {
        "raw": "../../"
    }
}
```

see e.g., https://github.com/bids-standard/bids-examples/issues/303 the `qmri_mp2rage` dataset, and turning `RawSources` there into `Sources` with BIDS-URIs

Reply:
@effigies any opinion?

Reply:
see suggestion https://github.com/bids-standard/bids-specification/pull/918/files#r927333853